pkgs/by-name/im/imager/package.nix

```
@@ -0,0 +1,98 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  gtk2-x11,
+  pkg-config,
+  python3,
+  gfortran,
+  lesstif,
+  cfitsio,
+  getopt,
+  perl,
+  groff,
+  which,
+  darwin,
+  ncurses,
+}:
+
+let
+  python3Env = python3.withPackages (
+    ps: with ps; [
+      numpy
+      setuptools
+    ]
+  );
+in
+
+stdenv.mkDerivation rec {
+  version = "4.4-01";
+  pname = "imager";
+
+  src = fetchurl {
+    url = "https://dave.oasu.u-bordeaux.fr:5000/5964614a-44f2-47fd-ba18-5f155543044e/imager-last.tgz";
+    sha256 = "sha256-cIgsj/OYzi8UZc9Gk/0bN3vHD8Kh+y1+EAPzM4J7t+4=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    groff
+    perl
+    getopt
+    gfortran
+    which
+  ];
+
+  buildInputs = [
+    gtk2-x11
+    lesstif
+    cfitsio
+    python3Env
+    ncurses
+  ] ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [ CoreFoundation ]);
```

No need to manually specify apple-sdk anymore
```suggestion
  ];
```